x/oracle/abci.go

```diff
@@ -63,14 +59,16 @@ func EndBlocker(ctx sdk.Context, k keeper.Keeper, experimental bool) error {
 			}
 
 			if experimental {
-				// Stamp rate every stamp period if asset is set to have historic stats tracked
-				if isPeriodLastBlock(ctx, params.StampPeriod) && params.HistoricAcceptList.Contains(ballotDenom.Denom) {
-					k.AddHistoricPrice(ctx, ballotDenom.Denom, exchangeRate)
+				// Stamp historic price if historic stamp period has passed
+				if isPeriodLastBlock(ctx, params.HistoricStampPeriod) {
+					k.AddHistoricPrice(ctx, strings.ToUpper(ballotDenom.Denom), exchangeRate)
```

Comment:
shouldn't this be normalized when the ballot is created?

Reply:
If `VotePeriod` == `HistoricStampPeriod` yes, but we decided on having `HistoricStampPeriod` be a parameter to be able to test whether or not prices need to be stamped as frequent as the `VotePeriod`.

Reply:
Initially yes but I changed it back to not convert to upper case to keep consistent with the `SetExchangeRate` method in the endblocker